pkg/receiver/jobreceiver/command/exec_linux.go

```diff
@@ -0,0 +1,15 @@
+package command
```

Comment:
For the platform specific `setOptions`, can we pick the right one at runtime instead of using build tags? Doesn't look like we use any platform-specific symbols.

Reply:
Afraid the syscall.SysProcAttr structure is os specific.

`for GOOS in "linux" "darwin" "windows"; go doc -short syscall.SysProcAttr`